reference/shaders/nvapi/ray-query-cluster-id.noglsl.nvapi.comp

```diff
@@ -0,0 +1,101 @@
+; SPIR-V
+; Version: 1.4
+; Generator: Unknown(30017); 21022
+; Bound: 65
+; Schema: 0
+OpCapability Shader
+OpCapability ImageBuffer
+OpCapability RayQueryKHR
+OpCapability RayTraversalPrimitiveCullingKHR
+OpCapability RayTracingClusterAccelerationStructureNV
+OpExtension "SPV_KHR_ray_query"
+OpExtension "SPV_NV_cluster_acceleration_structure"
+OpMemoryModel Logical GLSL450
+OpEntryPoint GLCompute %3 "main" %8 %12 %17 %21
+OpExecutionMode %3 LocalSize 64 1 1
+OpName %3 "main"
+OpDecorate %8 DescriptorSet 0
+OpDecorate %8 Binding 0
+OpDecorate %12 DescriptorSet 0
+OpDecorate %12 Binding 1
+OpDecorate %12 NonReadable
+OpDecorate %17 BuiltIn LocalInvocationIndex
+%1 = OpTypeVoid
+%2 = OpTypeFunction %1
+%5 = OpTypeInt 32 1
+%6 = OpTypeAccelerationStructureKHR
+%7 = OpTypePointer UniformConstant %6
+%8 = OpVariable %7 UniformConstant
+%9 = OpTypeInt 32 0
+%10 = OpTypeImage %9 Buffer 0 0 0 2 R32ui
+%11 = OpTypePointer UniformConstant %10
+%12 = OpVariable %11 UniformConstant
+%16 = OpTypePointer Input %9
+%17 = OpVariable %16 Input
+%19 = OpTypeRayQueryKHR
+%20 = OpTypePointer Private %19
+%21 = OpVariable %20 Private
+%22 = OpConstant %9 132
+%23 = OpConstant %9 170
+%24 = OpTypeFloat 32
+%25 = OpConstant %24 1
+%26 = OpConstant %24 5
+%27 = OpConstant %24 2
+%28 = OpConstant %24 6
+%29 = OpConstant %24 3
+%30 = OpConstant %24 7
+%31 = OpTypeVector %24 3
+%33 = OpConstant %24 4
+%35 = OpConstant %24 8
+%36 = OpTypeBool
+%39 = OpConstant %9 1
+%41 = OpConstant %9 0
+%46 = OpConstant %9 2
+%47 = OpTypeVector %9 4
+%3 = OpFunction %1 None %2
+%4 = OpLabel
+OpBranch %57
+%57 = OpLabel
+%13 = OpLoad %10 %12
+%15 = OpLoad %6 %8
+%18 = OpLoad %9 %17
+%32 = OpCompositeConstruct %31 %25 %27 %29
+%34 = OpCompositeConstruct %31 %26 %28 %30
+OpRayQueryInitializeKHR %21 %15 %22 %23 %32 %33 %34 %35
+%37 = OpRayQueryProceedKHR %36 %21
+OpSelectionMerge %61 None
+OpBranchConditional %37 %58 %61
+%58 = OpLabel
+OpBranch %59
+%59 = OpLabel
+%42 = OpRayQueryGetRayFlagsKHR %9 %21
```

Comment:
Okay, so because this is defined in HLSL as `#define NvRtGetCandidateClusterID(rq) __NvRtGetCandidateClusterID(rq.RayFlags())`, there's this `GetRayFlags` op that we don't actually need for anything. Ideally I'd suppress emitting it, but there's no such mechanism in dxil-spirv I could use… is there?